cryptography-engineer/src/ec_fft/README.md

```diff
@@ -4,18 +4,16 @@ Hi there! Welcome to the `ec_fft` test that you are about to take. We will guide
 
 #### Monomial Reference String
 
-Universal zk-SNARKs like PlonK need to run a one-time trusted setup ceremony to generate a Structured Reference String (SRS). Any number of participants can participate in this ceremony but only _one_ of all the participants need to be _honest_. As a part of this ceremony, each participant contributes to the setup with their own _secret_ which they are free to choose. Even if one of the participants generates this secret randomly and destroy it successfully, the setup ceremony is considered to be successful. The output of the setup ceremony is a structure reference string of the form:
+Universal zk-SNARKs like PlonK need to run a one-time trusted setup ceremony to generate a Structured Reference String (SRS). Any number of participants can participate in this ceremony but only _one_ of all the participants needs to be _honest_. As a part of this ceremony, each participant contributes to the setup with their own _secret_ which they are free to choose. If even one of the participants generates this secret randomly and destroys it successfully, the setup ceremony is considered to be successful. Fur our purposes, the output of the setup ceremony is a structure reference string of the form:
 
 $$
 \begin{aligned}
 \mathbb{G}_1 \text{ points: }
 \big([1]_1,[x]_1, [x^2]_1, [x^3]_1, \dots, [x^{N-1}]_1\big), \\
-\mathbb{G}_2 \text{ points: }
```

Comment:
No benefit to having this here if it's not used IMO.

Reply:
Right